### PR TITLE
Grammar Error Correction in Documentation

### PR DESCRIPTION
**Pull request by pidoxy**
resolves issue #11

### CODE REVIEW

**Review by Sammybams — CHANGES_REQUESTED** [review]
Thank you @pidoxy for your contribution. There are other grammatical errors that have not been corrected in this particular documentation. We would appreciate it if you could spot the rest and correct too. Thank you

**Comment by pidoxy** [comment]
This same contribution guide yeah?

Alright I'll go through it again

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ python -m venv AI-Hacktober-MLSA
 Activate the created virtual environment
 ```
 # Windows
-AI-Hacktober-MLSA\Scripts\activate
+AI-Hacktober-MLSA\scripts\activate
```
Comment (Sammybams) [inline CONTRIBUTING.md:31]:
```suggestion
AI-Hacktober-MLSA\Scripts\activate
```

**Review by Sammybams — APPROVED** [review]
Thank you very much @pidoxy. The other things you changed wasn't the actual error left but we greatly appreciate the contribution as it has helped us identify a better wording for some texts you changed.

**Comment by pidoxy** [comment]
What was the error left if you don't mind?

**Pull request merged**